yt_dlp/extractor/livestreamfails.py

```diff
@@ -0,0 +1,64 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import parse_iso8601
+
+
+class LivestreamFailsIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?livestreamfails\.com/(?:post|clip)/(?P<id>[0-9]+)'
+    _TEST = {
+        'url': 'https://livestreamfails.com/clip/18645',
+        'md5': '4438ab40c35a0df2553ae30f941d6cd4',
+        'info_dict': {
+            'id': '18645',
+            'ext': 'mp4',
+            'title': 'Zuckerberg gets Jebaited',
+            'thumbnail': r're:^https?://.*\.(jpe?g|png|webp)$',
+            'timestamp': 1523391367,
+            'upload_date': '20180410'
+        }
+    }
+
+    _DEFAULT_API_URL = 'https://api.livestreamfails.com'
+    _DEFAULT_VIDEO_SERVER = 'https://livestreamfails-video-prod.b-cdn.net'
+    _DEFAULT_IMAGE_SERVER = 'https://livestreamfails-image-prod.b-cdn.net'
+
+    def _extract_config_url(self, prop, script, name, default):
+        url = self._search_regex(fr'(["\']?){prop}\1:\s*(["\'])(?P<url>.+?)\2', script, name, fatal=False, group='url')
+
+        if url is None:
+            self.report_warning(f'Falling back to default {name}')
+            return default
```

Comment:
search_regex already issues warning
```suggestion
        url = self._search_regex(fr'(["\']?){prop}\1:\s*(["\'])(?P<url>.+?)\2', script, name, fatal=False, group='url') or default
```